virt/virt-web-console-overview.adoc

```diff
@@ -0,0 +1,274 @@
+====
+
+[id='virtualization-overview-top-consumers_{context}']
+=== Top consumers tab
+
+The *Top consumers* tab displays CPU, memory, and storage usage.
+
+You can filter the results by resource or virtual machine.
+
+.Top consumers image:icon-folder.png[tab icon,25]
+[%collapsible]
+====
+* CPU image:icon-page.png[page icon,25]
+* Memory image:icon-page.png[page icon,25]
+* Memory swap traffic image:icon-page.png[page icon,25]
+* vCPU wait image:icon-page.png[page icon,25]
+* Storage throughput image:icon-page.png[page icon,25]
+* Storage IOPS image:icon-page.png[page icon,25]
+====
+
+[id='virtualization-overview-migrations_{context}']
+=== Migrations tab
+
+The *Migrations* tab displays the state of virtual machine migrations during a configurable time period.
+
+.Migrations image:icon-folder.png[tab icon,25]
+[%collapsible]
+====
+* Time period image:icon-pencil.png[pencil icon,25]
+* VirtualMachineInstanceMigrations list image:icon-page.png[page icon,25]
+====
+
+[id='virtualization-overview-settings_{context}']
+=== Settings tab
+
+The *Settings* tab contains the following sections:
+
+* General image:icon-folder.png[tab icon,25]
+* Live migration image:icon-folder.png[tab icon,25]
+* Templates project image:icon-folder.png[tab icon,25]
+* User permissions image:icon-folder.png[tab icon,25]
+
+.Settings image:icon-folder.png[tab icon,25]
+[%collapsible]
+====
+
+[id='ui-virtualization-settings-general']
+.General image:icon-folder.png[tab icon,25]
+[%collapsible]
+=====
+* Service name
+* Provider
+* more settings
+=====
+
+[id='ui-virtualization-settings-live-migration']
+.Live migration image:icon-folder.png[tab icon,25]
+[%collapsible]
+=====
+* Max. migrations per cluster image:icon-pencil.png[pencil icon,25]
+* Max. migrations per node image:icon-pencil.png[pencil icon,25]
+* Live migration network image:icon-pencil.png[pencil icon,25]
+=====
+
+[id='ui-virtualization-settings-templates-project']
+.Templates project image:icon-folder.png[tab icon,25]
+[%collapsible]
+=====
+* Project  image:icon-pencil.png[pencil icon,25]
+=====
+
+[id='ui-virtualization-settings-user-permissions']
+.User permissions image:icon-folder.png[tab icon,25]
+[%collapsible]
+=====
+* List of user permissions
+=====
+
+====
+// end of Virtualization > Settings block
+
+[id='ui-virtualmachines-page_{context}']
+== VirtualMachines page
+
+The *VirtualMachines* page contains the following elements:
+
+* Virtualization dashboard image:icon-link.jpg[link icon,25]
+* Create VirtualMachine:
+
+  ** From form
+  ** From YAML
+
+* List of virtual machines
+
+Select and click a *VirtualMachine* to view the *VirtualMachine details* page.
+
+// Made this level 2 so that contents would appear in TOC
+[id='ui-virtualmachine-details-page_{context}']
+== VirtualMachine details page
+
+(Logically, this should be next level header. However, if we make this level 3, the tabs will not show up in the TOC. Suggestions or workarounds?)
```

Comment:
I just found this: https://docs.asciidoctor.org/asciidoc/latest/toc/levels/

"By default, the TOC displays level 1 (==) and level 2 (===) section titles. You can assign a different level depth with the toclevels attribute."

This might work:
```
:_content-type: ASSEMBLY
[id="virt-web-console-overview"]
include::_attributes/common-attributes.adoc[]
= {VirtProductName} web console
:context: virt-web-console-overview
:toclevels: 3

toc::[]
```